src/hotspot/share/compiler/compileBroker.cpp

```diff
@@ -3198,7 +3198,7 @@ void CompileBroker::print_times(bool per_compiler, bool aggregate) {
 
   if (StoreCachedCode || LoadCachedCode) { // Check flags because SC cache could be closed already
     tty->cr();
-    SCCache::print_timers_on(tty);
+    AOTCodeCache::print_timers_on(tty);
```

Comment:
There is a mention of "SC cache" a few lines above.

Reply:
Will fix